warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -1317,18 +1318,21 @@ protected TermFrequencyAggregator buildTermFrequencyAggregator(String identifier
      *            an existing EventDataQueryFilter
      * @return a ChainableEventDataQueryFilter
      */
+    @Deprecated
     protected ChainableEventDataQueryFilter createWrappedTermFrequencyFilter(String identifier, JexlNode node, EventDataQueryFilter existing) {
```

Comment:
Instead of deprecating, I would just get rid of this.

Reply:
Deprecating before removal allows for cleaner integration

Reply:
I would still just get rid of this.  It isn't like this is a public-facing api that everyone is going to be impacted by - this is internal code.

Reply:
very well, done and done